renderdocui/Windows/Dialogs/VirtualOpenFileDialog.cs

```diff
@@ -83,6 +83,8 @@ private string ChosenPath
         {
             get
             {
+                if (m_RM.Remote.Hostname.StartsWith("adb:"))
+                    return FileName;// Dont add a slash to Android package name
```

Comment:
Done. The capture tab now shows my package as "/com.package.name", but its not a big deal.